src/components/basic/Badge.tsx

```diff
@@ -27,13 +28,25 @@ const Comp = styled.span<{ isRead: boolean; isNumber: boolean }>`
 
 const Badge = forwardRef<HTMLElement, BadgeProps>(function BadgeFn({ type, value, ...rest }, ref) {
 	const isNumber = useMemo(() => typeof value === 'number', [value]);
-	const badgeText = useMemo(() => (isNumber && value > 99 ? '99+' : value), [value, isNumber]);
+	const badgeText = useMemo(() => (isNumber && value > 999 ? '999+' : value), [value, isNumber]);
```

Comment:
why not setting a constant to define the max value, and using it to do all checks and other things with this number? Something like ```value > MAX_VALUE ?  `${MAX_VALUE}+` : value  ```